metricbeat/etc/fields.yml

```diff
@@ -1447,62 +1464,21 @@
               description: >
                 Used memory.
 
-            - name: free
-              type: long
-              description: >
-                Available memory.
-
             - name: used.pct
               type: half_float
               description: >
                 The percentage of used memory.
 
-            - name: actual
-              type: group
+            - name: free
+              type: long
               description: >
-                Actual memory fields.
-              fields:
-                - name: used.bytes
-                  type: long
-                  description: >
-                    Actual used memory. This value is the "used" memory minus the memory used for disk caches and buffers.
-                    Available only on Unix.
-
-                - name: free
-                  type: long
-                  description: >
-                    Actual available memory. This value is the "free" memory plus the memory used for disk caches and
-                    buffers. Available only on Unix.
+                Free memory. For a human readable value use Available.
```

Comment:
"human readable" means to me that it's the same value formatted with units like MB or GB. It's like `df -h`.

Consider saying this "The total amount of free memory in bytes. This value does not include memory consumed by system caches and buffers (see `system.memory.available`)."